src/qml/TrackerSettings.qml

```diff
@@ -0,0 +1,546 @@
+        }
+
+        Label {
+          text: qsTr("Minimum distance [%1]").arg( UnitTypes.toAbbreviatedString( infoDistanceArea.lengthUnits ) )
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          enabled: minimumDistance.checked
+          visible: minimumDistance.checked
+          Layout.leftMargin: 8
+          Layout.fillWidth: true
+        }
+
+        QfTextField {
+          id: minimumDistanceValue
+          width: minimumDistance.width
+          font: Theme.defaultFont
+          enabled: minimumDistance.checked
+          visible: minimumDistance.checked
+          horizontalAlignment: TextInput.AlignHCenter
+          Layout.preferredWidth: 60
+          Layout.preferredHeight: font.height + 20
+
+          inputMethodHints: Qt.ImhFormattedNumbersOnly
+          validator: DoubleValidator { locale: 'C' }
+
+          onTextChanged: {
+            if( text.length === 0 || isNaN(text) ) {
+              positioningSettings.trackerMinimumDistance = NaN
+            } else {
+              positioningSettings.trackerMinimumDistance = parseFloat( text )
+            }
+          }
+        }
+
+        Label {
+          text: qsTr("Activate sensor constraint")
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          Layout.fillWidth: true
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: sensorCapture.toggle()
+          }
+        }
+
+        QfSwitch {
+          id: sensorCapture
+          Layout.preferredWidth: implicitContentWidth
+          Layout.alignment: Qt.AlignTop
+          checked: false
+          onCheckedChanged: {
+            positioningSettings.trackerSensorCaptureConstraint = checked
+          }
+        }
+
+        Label {
+          text: qsTr("Record when all active constraints are met")
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          Layout.fillWidth: true
+          enabled: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+          visible: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: allConstraints.toggle()
+          }
+        }
+
+        QfSwitch {
+          id: allConstraints
+          Layout.preferredWidth: implicitContentWidth
+          Layout.alignment: Qt.AlignTop
+          enabled: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+          visible: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+          checked: false
+          onCheckedChanged: {
+            positioningSettings.trackerMeetAllConstraints = checked
+          }
+        }
+
+        Label {
+          text: qsTr( "When enabled, vertices with only be recorded when all active constraints are met. If the setting is disabled, individual constraints met will trigger a vertex addition." )
```

Comment:
```suggestion
          text: qsTr( "When enabled, vertices will only be recorded when all active constraints are met. If the setting is disabled, individual constraints met will trigger a vertex addition." )
```

I think the first sentence is fixed, but the second one still does not make sense to me.

Reply:
Tried to improve by adding a word :) what it means is that say you have a time constraint of 5 sec and a distance constraint of 10 meters, a vertex will be added when individual constraints "turn green", i.e. when the time goes above 5 sec of when the distance goes above 10 meters.